interfaces/snap_app_set.go

```diff
@@ -32,7 +32,7 @@ func (a *SnapAppSet) SecurityTagsForConnectedPlug(plug *ConnectedPlug) ([]string
 // the plug.
 func (a *SnapAppSet) SecurityTagsForPlug(plug *snap.PlugInfo) ([]string, error) {
 	if plug.Snap.InstanceName() != a.info.InstanceName() {
-		return nil, fmt.Errorf("plug %s is from %s, expected snap: %s", plug.Name, plug.Snap.InstanceName(), a.info.InstanceName())
+		return nil, fmt.Errorf("plug %q is from snap %q, expected snap: %q", plug.Name, plug.Snap.InstanceName(), a.info.InstanceName())
```

Comment:
these errors should be prefixed with "internal error", also maybe:

"internal error: ... from snap %q, security tags can only be computed for processed target snap: %q" ?